core/src/banking_stage.rs

```diff
@@ -1283,20 +1271,14 @@ impl BankingStage {
     #[allow(clippy::too_many_arguments)]
     /// Process the incoming packets
     fn process_packets(
```

Comment:
nit: rename to something like `receive_and_buffer_packets` and update the above comment since this doesn't process anymore

Reply:
done